python/semantic_kernel/memory/volatile_memory_store.py

```diff
@@ -50,3 +49,43 @@ async def get_nearest_matches_async(
         # Then, take the top N results
         top_results = filtered_results[:limit]
         return top_results
+
+    async def compute_similarity_scores(
```

Comment:
I don't think this needs to be `async` (you aren't `await`ing anything in here)

Reply:
Done